Adding some warning for eternal orchestration

cfe84 opened the pull request:
Until https://github.com/Azure/azure-functions-durable-extension/issues/67 is fixed.

azurecla commented:
Hi __@DexterPOSH__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

azurecla commented:
__@DexterPOSH__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

PRmerger commented:
@DexterPOSH : Thanks for your contribution to the Azure documentation! The author, @mgoedtel, has been notified to review your proposed change.

MGoedtel commented:
@DexterPOSH thank you for taking the time to submit a request to make corrections/updates to this article.  I have reviewed them and approve them to be merged.

DexterPOSH commented:
Thanks @MGoedtel , I have few more changes coming your way for review.

MGoedtel commented:
#sign-off

PRmerger commented:
Only the author of the article can use the sign off comment to add the ready-to-merge label.

jomolnar commented:
@Matt Goedtel Hi Matt
Your sign off did not work with the comment automation because your GitHub account is listed incorrectly in the article metadata: authors="mgoedtel"
It needs to match your GitHub account EXACTLY, including capitalization. Please fix your GitHub account in all your articles after this is merged.

cfe84 committed:
Adding some warning for eternal orchestration

Until https://github.com/Azure/azure-functions-durable-extension/issues/67 is fixed.

cgillum reviewed (changes requested):
Thanks for submitting this PR!

A correction: It's not that calls to `ContinueAsNew` will be ignored. It's really that calls to `RaiseEventAsync` will be ignored.  Consider rephrasing to something like this:

> Currently any `RaiseEventAsync` messages sent to an orchestrator function may get dropped if they either have not been processed before the call to `ContinueAsNew`, or if they were sent before the new execution has started running. For example, if a function sends 10 messages to an orchestration using `RaiseEventAsync`, and the first messages received by the orchestrator function results in a call to `ContinueAsNew`, then the other 9 messages will be lost. Consider carefully whether this behavior might negatively affect you when deciding whether to use the `ContinueAsNew` API.
